src/content/docs/en/guides/content-collections.mdx

```diff
@@ -542,6 +542,23 @@ If you have an existing Astro project, such as a blog, that uses Markdown or MDX
  
 See how to convert a basic blog example from `src/pages/posts/` to `src/content/posts` in our [step-by-step tutorial](/en/tutorials/add-content-collections/) that uses the codebase from [the Build a Blog tutorial's finished project](https://github.com/withastro/blog-tutorial-demo).
 
+## Enabling Build Caching
+
+<p>
+  <Since v="3.5.0" />
+</p>
+
+If you are working with large Content Collections, it is possible to enable cached builds with the new [`experimental.contentCollectionCache`](/en/reference/configuration-reference/#experimentalcontentcollectioncache) flag. This experimental feature optimizes Astro's build process, enabling unchanged Content Collections to be stored and reused between builds.
```

Comment:
```suggestion
If you are working with large collections, you may wish to enable cached builds with the [`experimental.contentCollectionCache`](/en/reference/configuration-reference/#experimentalcontentcollectioncache) flag. This experimental feature optimizes Astro's build process, enabling unchanged collections to be stored and reused between builds.
```
nit: you can opt in even if you don't have large collections!  😄

Reply:
Great catch!